<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZeusOps Debug Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblInterval">
        <property name="text">
         <string>Interval (ms):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinInterval">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>15000</number>
        </property>
        <property name="singleStep">
         <number>250</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTargets">
        <property name="text">
         <string>Ping Targets:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtTargets">
        <property name="text">
         <string>8.8.8.8,ts.zeusops.com</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblArmaRunning">
        <property name="text">
         <string>ARMA3 running:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblArmaState">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblLogStatsLeft">
        <property name="text">
         <string>Logfile Size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblLogStats">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblShowLog">
        <property name="text">
         <string>Show Internal Log:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cboxShowLog">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblOutputDir">
        <property name="text">
         <string>Log Output Directory:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="edtOutputDir"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartStop">
      <property name="text">
       <string>Start Measurement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblLog">
      <property name="text">
       <string>Internal Log:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="logWidget"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
